tools/cli/go-whisk-cli/commands/rule.go

```diff
@@ -388,6 +388,19 @@ var ruleListCmd = &cobra.Command{
                     map[string]interface{}{"name": getClientNamespace(), "err": err})
             werr := whisk.MakeWskErrorFromWskError(errors.New(errStr), err, whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.NO_DISPLAY_USAGE)
             return werr
+        } else {
```

Comment:
This is the recent commit to apache master, I probably didn't do this right. I can back it out if you'd like